msteams-platform/TOC.yml

```diff
@@ -827,6 +827,9 @@
       - name: Build Adaptive Card-based Loop component
         href: m365-apps/cards-loop-component.md
         displayName: loop component, weburl, metadata
+  - name: Add an Outlook Add-in to a Teams app
+    href: m365-apps/combine-office-add-in-and-teams-app.md
+    displayName: Microsoft 365, Office 365, Outlook add-in
```

Comment:
SEO suggestion that can be added: integrate Outlook Add-in to Teams, Teams app Outlook Add-in